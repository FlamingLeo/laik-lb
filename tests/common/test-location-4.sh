#!/bin/sh
${LAUNCHER-./launcher} -n 4 ../src/locationtest | LC_ALL='C' sort > test-location-4.out
#cmp test-location-4.out "$(dirname -- "${0}")/test-location-4.expected"
