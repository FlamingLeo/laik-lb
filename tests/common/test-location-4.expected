Testing identifiers - world index 0, location 'x260:74236'
Testing identifiers - world index 1, location 'x260:74237'
Testing identifiers - world index 2, location 'x260:74239'
Testing identifiers - world index 3, location 'x260:74238'
at 0: identifier for ID 0 is 'x260:74236'
at 0: identifier for ID 1 is 'x260:74237'
at 0: identifier for ID 2 is 'x260:74239'
at 0: identifier for ID 3 is 'x260:74238'
at 1: identifier for ID 0 is 'x260:74236'
at 1: identifier for ID 1 is 'x260:74237'
at 1: identifier for ID 2 is 'x260:74239'
at 1: identifier for ID 3 is 'x260:74238'
at 2: identifier for ID 0 is 'x260:74236'
at 2: identifier for ID 1 is 'x260:74237'
at 2: identifier for ID 2 is 'x260:74239'
at 2: identifier for ID 3 is 'x260:74238'
at 3: identifier for ID 0 is 'x260:74236'
at 3: identifier for ID 1 is 'x260:74237'
at 3: identifier for ID 2 is 'x260:74239'
at 3: identifier for ID 3 is 'x260:74238'
